Scripts/Tracking/Follow/Modifier/TargetsFollowSource.cs

```diff
@@ -18,50 +18,38 @@ public class TargetsFollowSource : FollowModifier
         /// </summary>
         /// <param name="source">The source <see cref="Transform"/> to utilize in the modification.</param>
         /// <param name="target">The target <see cref="Transform"/> to modify.</param>
-        public override void UpdatePosition(Transform source, Transform target)
+        protected override void DoUpdatePosition(Transform source, Transform target)
         {
-            if (!isActiveAndEnabled || appliedModifier == null)
+            if (appliedModifier != null)
             {
-                return;
+                appliedModifier.UpdatePosition(target, source);
             }
-
-            appliedModifier.UpdatePosition(target, source);
-            CachedSource = appliedModifier.CachedSource;
-            CachedTarget = appliedModifier.CachedTarget;
         }
 
         /// <summary>
         /// Updates the target <see cref="Transform"/> rotation using the source <see cref="Transform"/>.
         /// </summary>
         /// <param name="source">The source <see cref="Transform"/> to utilize in the modification.</param>
         /// <param name="target">The target <see cref="Transform"/> to modify.</param>
-        public override void UpdateRotation(Transform source, Transform target)
+        protected override void DoUpdateRotation(Transform source, Transform target)
         {
-            if (!isActiveAndEnabled || appliedModifier == null)
+            if (appliedModifier != null)
             {
-                return;
+                appliedModifier.UpdateRotation(target, source);
             }
-
-            appliedModifier.UpdateRotation(target, source);
-            CachedSource = appliedModifier.CachedSource;
-            CachedTarget = appliedModifier.CachedTarget;
         }
 
         /// <summary>
         /// Updates the target <see cref="Transform"/> scale using the source <see cref="Transform"/>.
         /// </summary>
         /// <param name="source">The source <see cref="Transform"/> to utilize in the modification.</param>
         /// <param name="target">The target <see cref="Transform"/> to modify.</param>

```

Comment:
This XML docs block should just be `<inheritdoc />`.

Reply:
The docs are different for this method